content/documentation/overview/index.adoc

```diff
@@ -46,56 +46,56 @@ image::graph-view.png[Demo Website]
 
 === Applications
 
-In the applications tab we can see all the apps running in our environment.
+The Applications menu shows all the apps running in our environment.
 
 [#img-kiali-app-list]
 .List of Applications
 image::app-list.png[Demo Website]
 {nbsp} +
-Kiali provide us all the information related with the application like the health.
+Kiali provides detailed information related to the application, such as its health or the list of its workloads.
 {nbsp} +
 [#img-kiali-app-view]
-.Info of Application
+.Application Info
 image::app-view-info.png[Demo Website]
 {nbsp} +
-We can also see the metrics associated with an application provided by grafana
+We can also see the metrics associated with an application provided by Istio.
 {nbsp} +
 [#img-kiali-metrics]
-.Metrics of Application
+.Application Metrics
 image::app-metrics.png[Demo Website]
 {nbsp} +
 
 === Workloads
 
-In the workload tab we have the list of workloads with their health, error rate and labels validations.
+The Workloads menu shows the list of workloads with their health, error rate and labels validations.
 {nbsp} +
 [#img-kiali-workload-list]
 .List of Workloads
 image::workload-list.png[Demo Website]
 {nbsp} +
-When we choose a workload we can see the information related to it and associated pods and services.
+By selecting a workload, the related information is displayed along with the associated pods and services.
 {nbsp} +
 [#img-kiali-workload-view]
-.List of Workloads
+.Workload Info
 image::workload-view-pods.png[Demo Website]
 {nbsp} +
 
 === Services
 
-In the service tab we go to the page detail of a service, we have a health icon to check if our service has some problem like error rate.
+The Services menu shows the list of services, with the health icon to check indicators such as the error rate.
 
-In this page we have an overview about the service : labels, type, ip , ports, endpoints, workloads, destiantion rules, virtual services...
+When selecting a single service, the details page includes service labels, type, ip, ports, endpoints, workloads, destination rules, virtual services...
```

Comment:
the details page -> its details page
and  "..." should be replaced with something more concrete. "And more details/properties"